docs/playroom/snippets.ts

```diff
@@ -639,9 +639,13 @@ const snippets = [
 		name: 'Basic',
 		code: `<ProgressIndicator
 			items={[
-				{ href: '#', label: 'Introduction', status: 'doing' },
-				{ href: '#', label: 'Business Contacts', status: 'todo' },
-				{ href: '#', label: 'Case Studies', status: 'done' },
+				{ label: 'Introduction', status: 'done', href: '#', isActive: true },
+				{ label: 'Submit evidence', status: 'saved', href: '#' },
+				{ label: 'Organisations', status: 'started', href: '#' },
+				{ label: 'Business contacts', status: 'error', href: '#' },
+				{ label: 'Case studies', status: 'todo', href: '#' },
+				{ label: 'Attachments', status: 'doing', href: '#' },
```

Comment:
I don't think we should be promoting this, and use TypeScript to deprecate the `doing` status

Reply:
It sounds like you want to promote the pattern of doing `status: 'started'` with `isActive: true` applied to it and leave the 'doing' status out of all of our documentation examples.

```js
[
    { label: 'Introduction', status: 'done', href: '#' },
    { label: 'Submit evidence', status: 'saved', href: '#' },
    { label: 'Organisations', status: 'started', href: '#', isActive: true },
    { label: 'Business contacts', status: 'error', href: '#' },
    { label: 'Case studies', status: 'todo', href: '#' },
    { label: 'Review and submit', status: 'blocked', href: '#' },
]
```

TS can't deprecate strings. It can only deprecate variables, props, and functions.

I wanted to add `isActive: true` to the playground example so that people are aware of it.

Reply:
I can't deprecate but I could do `console.warn('The "doing" status is deprecated. Use status "started" with "isActive: true" applied instead.')`

Reply:
I actually throw errors now if they add `isActive` to properties that don't support active state.

Reply:
Sounds like a good idea. Might be worth adding the that `doing` is deprecated in the docs also.

Reply:
I've added the deprecation warning... however I'm questioning if it is actually the right move to deprecate the "doing" status

Reply:
Applying a "doing" status for managing active state is easier than adding isActive:true to a started status.
 
The client would already have logic for managing status, it would likely require extra logic to manage an isActive property.
 
I started removing all mentions of "doing" from the documentation and it seems like we are making things more difficult for developers by moving towards `isActive:true` as a required property on the active item

Reply:
Message from Matt over MS Teams:
> doing may be easier, but it's not scalable. We've identified that end users need to know where they're on one of these others steps, so our app devs already have to suit the scenarios where someone clicks on a "Cannot start" status. If they need to apply an isActive prop here, it's not much of a stretch to get them to also do that for started ones